test/clj/rems/test_browser.clj

```diff
@@ -691,6 +691,8 @@
       (is (btu/eventually-visible? {:tag :h1 :fn/has-text "test-handling"})))
     (testing "handler should see the applicant info"
       (btu/scroll-and-click :applicant-info-collapse-more-link)
+      (Thread/sleep 1000) ;; figure out what to wait for
+      (btu/screenshot "after-opening-applicant-info.png")
```

Comment:
something caused a race condition in CI, as the screenshot without `(Thread/sleep 1000)` showed a partially rendered form. i managed to replicate the test breakage locally by running the entire browser test suite, which co-incidentally broke also two other tests. these tests run completely fine in isolation, and apparently CI has enough beef to run them without failing as well.